tcf_website/static/course/javascript/instr_data.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Dropdown data population for the course-instructor page
+ * Author: j-alicia-long, 12/13/2020
+*/
+
+// Executed when DOM is ready
+jQuery(function($) {
+
+    /* Fetch instructor data for given course */
+    var pageSize = "1000";
+    var instrEndpoint = `/api/instructors/?course=${courseID}` +
```

Comment:
Nitpick but did you split it onto two lines to shorten line length?

Reply:
yep, my guideline is 80 chars max on a line. what do you think?

Reply:
That's fine